bpf/tc/attach.go

```diff
@@ -441,7 +434,7 @@ func CleanUpJumpMaps() {
 		if err != nil {
 			return err
 		}
-		if info.IsDir() && tcDirRegex.MatchString(info.Name()) {
+		if info.IsDir() && info.Name() != "globals" {
```

Comment:
Can we restrict to only the dirs that we expect?  I.e. the old regex unioned with whatever new dirs you're expecting?  don't want to break other apps that might have a dir in here.